ModelCatalogueGenomicsPlugin/grails-app/services/modelcataloguegenomicsplugin/GelJsonService.groovy

```diff
@@ -0,0 +1,160 @@
+package modelcataloguegenomicsplugin
+
+
+import grails.transaction.Transactional
+import org.modelcatalogue.core.CatalogueElement
+import org.modelcatalogue.core.DataClass
+
+
+//@Transactional
+class GelJsonService {
```

Comment:
Adam & I had already discussed this and agreed it would be better. But not right now.